gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -440,6 +440,9 @@ class BaselineIdea extends AbstractBaselinePlugin {
                 <name>autovalue.shaded</name>
                 <name>org.inferred.freebuilder.shaded</name>
                 <name>org.immutables.value.internal</name>
+                <name>com.palantir.conjure.java.client.config.ImmutablesStyle</name>
+                <name>com.palantir.sls.versionsImmutablesStyle</name>
```

Comment:
shouldn't this be `com.palantir.sls.versions.ImmutablesStyle`?

Reply:
Oops, fixed